### DISCUSSION TITLE
My custom domain always redirected too many times

### DISCUSSION THREAD

**Discussion by Folyd** [vercel/community#3308] [Help] [answered]
My custom domains ([rust.extension.sh](https://rust.extension.sh) and [go.extension.sh](https://go.extension.sh)) in the error state in the last two days. I confirm my DNS records were configured correctly, but I don't know why it not working. BTW, my domain Nameserver is Cloudflare. Could you please help me? Thanks a lot.

**Comment by Folyd** [comment, marked as answer]
Fixed. In case someone occurred the same problem, please refer to this link: https://community.cloudflare.com/t/community-tip-fixing-err-too-many-redirects/42335

**Reply by akshaysmurthy** [reply]
@Folyd What did you end up doing to solve this? I see there are quite a few ideas in the link you shared.

**Comment by sayhicoelho** [comment]
Had the same issue with CloudFlare. Go to SSL/TLS page and then set encryption mode to Full. 

![image](https://github.com/vercel/vercel/assets/8188365/ff22de6c-815e-4ec9-8058-fd40ef75e092)
